feature-pack-build-maven-plugin/src/main/java/org/wildfly/build/plugin/FeaturePackBuildMojo.java

```diff
@@ -65,13 +65,13 @@
     /**
      * The configuration file used for feature pack.
      */
-    @Parameter(alias = "config-file", defaultValue = "feature-pack-build.xml", property = "wildfly.feature.pack.configFile")
+    @Parameter(alias = "config-file", defaultValue = "${basedir}/feature-pack-build.xml", property = "wildfly.feature.pack.configFile")
     private String configFile;
```

Comment:
Yes, I can see how it woks in the original plugin and I tried to explain in the PR description why it is not good like that: we need a way to read `subsystems.xml` & Co. from under `target` directory.

Changing the interpretation of `configDir` to become the root of `subsystems.xml` & Co. only was the cleanest way I could see.

Yes, it can also be reached by introducing a new config param as @stuartwdouglas proposes, but I find that option much more dirty than the present one. I suppose that there is just a handful of projects using the build plugins and I doubt anybody has ever overridden the `configDir` default. I admit you may have more accurate info :)

So, if @stuartwdouglas still insists on a new config param, I am ready to go that way.